### PR TITLE
WindowSlideMax--- Python

### LINKED ISSUE #1750: Sliding Window Maximum in java
**Is your feature request related to a problem? Please describe.**
Sliding Window Maximum - Function to find and print the maximum element in every window of size k.

**Describe the solution you'd like**
A simple and concise approach which is implemented using Stack, it is somewhat as finding max on the right.

**Do you want to work on it**
Yes, I am a GSSOC'21 participant I would like to solve it in Java.

### PR DESCRIPTION
**Pull request by U-c0de**
Fixes #1750 

Do Check @tarun26091999 @Rohan27s @plazzy99 @yasharth291

### CODE REVIEW

**Comment by U-c0de** [comment]
Do Check @tarun26091999 @Rohan27s @plazzy99 @yasharth291

**Comment by U-c0de** [comment]
Do Check @tarun26091999 @Rohan27s @plazzy99 @yasharth291

**Comment by U-c0de** [comment]
Do Check @tarun26091999 @Rohan27s @plazzy99 @yasharth291

**Comment by U-c0de** [comment]
@Rohan27s @yasharth291  check it

File: Code/Python/SlidingWindowMax.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+2. if N > K
+
+N= 5
+K= 3
+lst= { 5 2 -1 0 3 }
+
+Output : 6
+
+
+Time Complexity : O(N)
```
Comment (Rohan27s) [inline Code/Python/SlidingWindowMax.py:58]: What is N here(mention it)

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by U-c0de** [comment]
@Rohan27s @yasharth291 check it

**Comment by Rohan27s** [comment]
resolve the conflicts

**Comment by U-c0de** [comment]
Review it @Rohan27s

File: Algorithm/Number_theory/readme.md [hunk 2]
```diff
@@ -48,5 +48,5 @@
 - Modular Exponentiation ----> [C++](/Code/C++/modular_exponentiation.cpp)
 - Palindrome Number ---> [C++](/Code/C++/palindrome_number.cpp) | [Python](/Code/Python/palindrome_number.py)
 - Taylor's Series of e^x ----> [Java](/Code/Java/Taylorseries.java)
-- To find all prime number from 2 to N ----> [C++](/Code/C++/Sieves_prime.cpp)
-
+- To find all prime number from 2 to N ----> [C++](Code/C++/Sieves_prime.cpp)
```
Comment (Rohan27s) [inline Algorithm/Number_theory/readme.md:51]: add / before Code in  To find all prime number from 2 to N ----> [C++](Code/C++/Sieves_prime.cpp)

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Review by Rohan27s — APPROVED** [review]

**Pull request merged**